src/components/app.tsx

```diff
@@ -24,41 +25,46 @@ export class AppComponent extends BaseComponent<{}, {}> {
 
   public render() {
 
-    const {ui} = this.stores;
-
-    const pictureAreaStyle: React.CSSProperties = {
-      position: "relative",
-      margin: 5,
-    };
+    const {ui, appMode} = this.stores;
 
     return (
-      <div className="app">
-        <div className="controls">
-          <SimulationControls>
-            <ControlArea />
-          </SimulationControls>
-        </div>
-        {ui.displayMode === "Simulation" &&
-          <div className="section simulation">
-            <div style={pictureAreaStyle}>
-              <SizeMe>
-                { ({size}: ISize ) => <PictureArea width={size.width ? size.width : 600} /> }
-              </SizeMe>
-            </div>
-          </div>
+      <div className="app-container">
+        {appMode !== "embed" &&
+          <div className="top-bar">DAT Cross Dam</div>
         }
-        {ui.displayMode === "Graph" &&
-          <div className="section chart">
-            <ChartDisplay />
+        <div className="controls-and-content-container">
+          {appMode === "dev" &&
+            <div className="left-panel">
+              <div className="controls">
+              <SimulationControls>
+                <ControlArea />
+              </SimulationControls>
+              </div>
+            </div>
+          }
+        <div className="main-content">
+            <div className="section simulation">
+              <SizeMe monitorHeight={true}>
+                {({ size }: ISize) =>
+                  <PictureArea width={size.width ? size.width : 0} height={size.height ? size.height : 0} />
```

Comment:
This might be worth a discussion. I choose, specifically, to not pass the height to the display, aka, the `PictureArea`, because, to my thinking, the display area must be always rendered with the same aspect ratio. Ergo, only passing in the width, using that as a ratio of the current-display width to the design width (of 600px) would then be used to scale in both the X and Y directions.

But, of course, I could be argued out of it! :)

Reply:
This needs to be resolved, but in the interest of unblocking further progress we'll get this merged in asap and continue fixing aspect ratios in the next PR.